### PR TITLE
Cyblethreatintel updates

### PR DESCRIPTION
**Pull request by cyble-dev**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by israelpoli** [comment]
Hi @cyble-dev
Thank you for contributing!
Apologies for the delay in reviewing your PR, I will review your code as soon as possible.

Thanks again

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:4]:
```suggestion
from typing import *
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:66]:
```suggestion
                fields_found.extend(result)
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:70]: Are you sure the item is not a list?

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:78]:
```suggestion
    def build_indicators(self, args: Dict[str, Any], data: list):
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:112]: In both options you lose all values except the last ones,
Are you sure this should be the case?

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:131]:
```suggestion
    def get_taxii(self, args: Dict[str, Any]):
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:158]: When a limit is not provided, the condition will not be met.
```suggestion
                if count == arg_to_number(args.get('limit', 1)):
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:192]:
```suggestion
def get_test_response(client: Client, args: Dict[str, Any]):
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:214]:
```suggestion
def get_feed_collection(client: Client):
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:229]:
```suggestion
def cyble_fetch_taxii(client: Client, args: Dict[str, Any]):
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:256]:
```suggestion
def fetch_indicators(client: Client):
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:288]:
```suggestion
def validate_input(args: Dict[str, Any]):
```

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:327]: What do you need it for?
You handle it within the Client

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:329]: There seems to be no use for it

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.yml
Comment (israelpoli) [inline Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.yml:136]: After the docker's PR is approved, the updated docker should be placed here

**Review by israelpoli — CHANGES_REQUESTED** [review]
Hi @cyble-dev 
Thank you for your contribution.

Looks good
Please see the comments I wrote
Some of the comments are a recommendation only

Thanks

**Comment by israelpoli** [comment]
The approval of this [PR](https://github.com/demisto/dockerfiles/pull/8152) is required before the approval of the current PR

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:327]: passing all the parameters at once via this

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:288]: updating

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:256]: updating

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:214]: updating

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:192]: updating

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:131]: updating

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:78]: updating

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:70]: yes

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:66]: updating

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (israelpoli) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:4]: It is used to declare a type in the signature of the functions, for example: `Dict[str, Any]` in line 288
`Dict` is in the `typing` library

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py]: updated

File: Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py
Comment (cyble-dev) [reply Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py:327]: will retain this

**Pull request merged**
